### ISSUE TITLE
Using Additional metrics to evaluate, the results are all the same, such as F1, ACC.

### ISSUE THREAD

**Issue by helloeng** [ThilinaRajapakse/simpletransformers#1396]
**Describe the bug**
When I call additional metrics to evaluate the results, the results of other metrics are exactly the same. What's the problem?

**To Reproduce**

from sklearn.metrics import f1_score, accuracy_score,precision_score,recall_score

def f1_multiclass(labels, preds):
      return f1_score(labels, preds, average='micro')

def precision_multiclass(labels, preds):
      return precision_score(labels, preds, average='micro')

def recall_multiclass(labels, preds):
      return recall_score(labels, preds, average='micro')

result, model_outputs, wrong_predictions = model.eval_model(eval_df, f1=f1_multiclass, acc=accuracy_score,precision=precision_multiclass,recall=recall_multiclass)

**Expected behavior**
Each metric value should be different.

**Screenshots**
![image](https://user-images.githubusercontent.com/64872312/163909409-e4eb96b7-413b-4df1-8cc5-f3bc54b65ced.png)
![image](https://user-images.githubusercontent.com/64872312/163909536-ce6c41ae-03b7-481d-a825-9f911bdc007d.png)

**Desktop (please complete the following information):**
 - ubuntu 20.04.1

**Additional context**
This task is a four-class text classification.
